x-pack/plugins/security/server/routes/api_keys/create.test.ts

```diff
@@ -42,6 +42,7 @@ describe('Create API Key route', () => {
   });
 
   describe('failure', () => {
+    test.todo('actually exercise different types of payload validation');
```

Comment:
Should this be implemented as part of this PR?

Reply:
I mentioned adding this TODO in my comment above: https://github.com/elastic/kibana/pull/130918#discussion_r876926374

Our existing unit tests for routes aren't structured in a way where the payload validation can be exercised. 

Seeing as we do already have an integration test that creates an API key with Kibana privileges, I think we should enhance the unit tests in a follow-on task. I added an issue for it here: #132670